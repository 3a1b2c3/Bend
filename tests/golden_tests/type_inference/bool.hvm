Not (True) = False
Not (False) = True

And (True) b = b
And (False) b = False