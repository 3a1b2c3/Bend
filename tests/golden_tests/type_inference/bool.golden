Adts: [
  0 => [(True), (False)]
]
Types: [
  Not => [Adt(0)]
  And => [Adt(18446744073709551615), Any]
]